ucithesis.cls

```diff
@@ -252,7 +252,7 @@
 	  \clearpage
 	\fi
 	\phantomsection
-	\addcontentsline{toc}{chapter}{ABSTRACT OF THE \uppercase{\Documenttitle}}
+	\addcontentsline{toc}{chapter}{ABSTRACT OF THE \texorpdfstring{\uppercase{\Documenttitle}}{DISSERTATION}}
```

Comment:
The alternative string only works for PhD though? What about MSc? Maybe add at least a comment that this may need updating?